docs/how-to-monitor-minio-with-prometheus.md

```diff
@@ -33,35 +33,42 @@ The Prometheus monitoring server
 
 Refer [Prometheus documentation](https://prometheus.io/docs/introduction/first_steps/) for more details.
 
-### 2. Prometheus configuration
+### 2. Generate Prometheus Config with mc
 
-Prometheus configuration is written in YAML. Add MinIO server details to the config file under `scrape_configs` section
+As Prometheus endpoint in MinIO is authorized. For a secured access using a bearer token approach, override the default prometheus config with the one generated using mc.
+To generate a prometheus config for an alias, use [mc](https://docs.min.io/docs/minio-client-quickstart-guide) as follows `mc admin prometheus generate <alias>`.
```

Comment:
Capitalization is not consistent here, if referring to the product should be capitalized, vs when added as part of a command (in this example, `...generate a Prometheus config` vs `mc admin prometheus`.